forge-ai/src/main/java/forge/ai/simulation/SpellAbilityPicker.java

```diff
@@ -147,8 +147,8 @@ private void printPlan(Plan plan, String intro) {
 
     private static boolean isSorcerySpeed(SpellAbility sa, Player player) {
         // TODO: Can we use the actual rules engine for this instead of trying to do the logic ourselves?
-        if (sa instanceof PlayLandAbility) {
-            return false;
+        if (sa instanceof LandAbility) {
```

Comment:
I think your changes make sense

Though when I tried your new test on master the planned decisions were also in the correct order so I'm not sure if it catches what it's supposed to?

Reply:
Let me take a look at that, thanks for taking the time to patch it in and try it!

Reply:
looks fine to me, lets merge this.